src/common/security/admiral/context.go

```diff
@@ -71,19 +75,20 @@ func (s *SecurityContext) IsSolutionUser() bool {
 
 // Can returns whether the user can do action on resource
 func (s *SecurityContext) Can(action rbac.Action, resource rbac.Resource) bool {
```

Comment:
can we set false always for admiral?

Reply:
To keep admiral works as before although we do not need it now. Let's remove it at #9236